build.sbt

```diff
@@ -8,7 +8,7 @@ scalaVersion := "2.13.14"
 scalacOptions ++= Seq("-release:11")
 crossScalaVersions := Seq(scalaVersion.value, "2.12.19")
 
-//releaseVersion := ReleaseVersion.fromAggregatedAssessedCompatibilityWithLatestRelease().value
+releaseVersion := ReleaseVersion.fromAggregatedAssessedCompatibilityWithLatestRelease().value
```

Comment:
We can put this line back now that the release of [v0.6.9](https://github.com/guardian/dynamo-db-switches/releases/tag/v0.6.9) succeeded!

Once v0.6.9 is showing up in https://index.scala-lang.org/guardian/dynamo-db-switches/artifacts/dynamo-db-switches?stable-only=true it will be ready for download, and you could even do another release just to check everything is still working correctly.